settings.gradle.kts

```diff
@@ -1,2 +1,7 @@
+pluginManagement {
+    includeBuild("build-logic")
+}
+
 rootProject.name = "Argos"
-include(":ASTProcessor")
```

Comment:
Why did you remove `ASTProcessor`? Since it is still in the project, would it be useful to have `ASTProcessor` as normal directories or why do you still keep it in the folders but remove it here?

Reply:
I have moved AST Processor inside the code-analyzer module. This was done following our architecture guidelines.
![image.png](https://files.reviewableusercontent.io/images/reviewable/64939916/stQq9PfHW4layxp2ExQDTVTC/image.png)
<!-- Reviewable comment -OCOITO03Df39s9YzO2N:b-wvhzcw -->
<!-- Sent from Reviewable.io -->